src/ctype/CTypeUtils.ts

```diff
@@ -6,41 +6,38 @@ import Ajv from 'ajv'
 import { CTypeModel, CTypeInputModel } from './CTypeSchema'
 import CType from './CType'
 import ICType from '../types/CType'
-import Crypto from '../crypto/index'
-
-export function verifyClaimStructure(claim: any, schema: any): boolean {
-  if (!verifySchema(schema, CTypeModel)) {
-    throw new Error('CType does not correspond to schema')
-  }
-  return verifySchema(claim, schema)
-}
-
-export function verifySchema(model: any, metaModel: any): boolean {
-  return verifySchemaWithErrors(model, metaModel)
-}
+import Crypto from '../crypto'
 
 export function verifySchemaWithErrors(
   model: any,
   metaModel: any,
   messages?: string[]
 ): boolean {
-  const ajv = new Ajv({
-    meta: false,
-  })
-  ajv.addMetaSchema(require('ajv/lib/refs/json-schema-draft-07.json'))
+  const ajv = new Ajv()
   ajv.addMetaSchema(CTypeModel)
   const result = ajv.validate(metaModel, model)
   if (!result && ajv.errors) {
-    ajv.errors.map((error: any) => {
-      if (messages) {
+    if (messages) {
+      ajv.errors.forEach((error: any) => {
         messages.push(error.message)
```

Comment:
Not purely part of this PR, so that's nitpicking / just a question:

Why not using functional instead?
```javascript  
if (messages) {
  messages = [...messages, ...ajv.errors.map(e => e.message)]; 
}
```

Reply:
This will not work, since we don't want to reassign parameters.
I will make a ticket, where we will change the return type of this function, to return either true, or messages.

Reply:
https://github.com/KILTprotocol/sdk-js/issues/151